ADavidson02/funny-dev
# final clean up of code

## Is this a fix or a feature?
* Fix

## What does it fix?
* This PR cleans up all components

## How should this be tested?
* Run the test suite then start the site locally along with the Funny-dev API to ensure they are both still working 

@ADavidson02
